From 15fd5826a8427d720b6145d20681825c6c0a0a02 Mon Sep 17 00:00:00 2001
From: Mansive <33560917+Mansive@users.noreply.github.com>
Date: Mon, 5 Jan 2026 18:56:10 -0600
Subject: [PATCH] Ignore libfrida files

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index 0be76d85cd..2e50a5fded 100644
--- a/.gitignore
+++ b/.gitignore
@@ -63,3 +63,5 @@ artifacts
 *.AppImage*
 /install*
 vulkansdk*.exe
+
+libfrida*
\ No newline at end of file
